### PR TITLE
Drop references to AngularDart

### PR DESCRIPTION
**Pull request by kevmoo**

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -15,7 +15,6 @@ port: 4000
 
 dart-site: '' 
 prev-url: https://v1-dartlang-org.firebaseapp.com
```
Comment (kevmoo) [inline _config.yml:17]: FYI @kwalrath – `prev-url` does not see to be referenced anywhere

File: src/_guides/testing.md [hunk 2]
```diff
@@ -17,16 +17,6 @@ using the [`dart test`][] command
 [`dart test`]: /tools/dart-test
 [`flutter test`]: {{site.flutter}}/docs/reference/flutter-cli
 
-{{site.alert.info}}
```
Comment (kevmoo) [inline src/_guides/testing.md:20]: This clarification seems superfluous now!

**Review by parlough — COMMENTED** [review]
I'll leave the approval on this to @kwalrath, but all the changes look good to me. Removing the mentions from Effective Dart especially makes sense. 

As for the build failing, if you rebase or merge in `master` you should be set as I recently accounted for the new cases of `unnecessary_type_check` in https://github.com/dart-lang/site-www/commit/208884ef0fcd5a6cd492089a02ead290d5fd1438

**Comment by kevmoo** [comment]
The failure here seems due to the new `unnecessary_type_check` – at least in part.

**Comment by kwalrath** [comment]
> The failure here seems due to the new `unnecessary_type_check` – at least in part.

I think this was fixed in #3590.

File: _config.yml (hunk 1 above)
Comment (kevmoo) [reply _config.yml:17]: done!

File: src/_guides/testing.md [hunk 3]
```diff
@@ -112,10 +102,6 @@ applications:
 * [package:webdriver]({{site.pub-pkg}}/webdriver)<br>
   A Dart package for interfacing with
   [WebDriver](https://www.w3.org/TR/webdriver/) servers.
```
[the remark this answers is not in the document]
Comment (kevmoo) [reply src/_guides/testing.md:104]: webdriver isn't well maintained. Dropping it!

**Comment by kevmoo** [comment]
@kwalrath – PTAL?

**Review by kwalrath — APPROVED** [review]
LGTM!

(I removed a leftover reference to the Angular docs)

**Pull request merged**